dependencies.yml

```diff
@@ -462,11 +462,14 @@ org.jetbrains.kotlinx:
 org.jlleitschuh.gradle:
   ktlint-gradle: { version: '9.2.1' }
 
+org.jooq:
+  joor: { version: '0.9.13' }
+
 org.jsoup:
   jsoup: { version: '1.13.1' }
 
-org.jooq:
-  joor: { version: '0.9.13' }
+org.junit-pioneer:
+  junit-pioneer: { version: '0.6.0' }
```

Comment:
Maybe just enough to run in a loop? What other features does it have we can leverage in the future?

Reply:
I also thought that adding this dependency is overkill a bit.
https://junit-pioneer.org/docs/range-sources/ seems useful, but we can add it later when we really need it.